txp/Pos/Txp/Toil/Utxo/Functions.hs

```diff
@@ -44,6 +44,7 @@ data VTxContext = VTxContext
     { -- | Verify that script versions in tx are known, addresses' and
       -- witnesses' types are known, attributes are known too.
       vtcVerifyAllIsKnown :: !Bool
+    , vtcAdoptedVersion   :: !ScriptVersion
```

Comment:
Can you rename it to `vtcAdoptedScriptVersion` please? There are many different versions which can be adopted, not only script version.